docs/src/ScrollableContainer.doc.js

```diff
@@ -325,26 +335,45 @@ function ScrollableContainerExample() {
   const [showModal, setShowModal] = React.useState(false);
   const [open, setOpen] = React.useState(false);
   const [selected, setSelected] = React.useState(null);
+  const [parentComponent, setParentComponent] = React.useState('modal');
   const anchorDropdownRef = React.useRef(null);
   const handleSelect = ({ item }) => {
     setSelected(item);
   };
 
+  const MODAL_Z_INDEX = new FixedZIndex(11)
+  const ANCHORED_Z_INDEX = new CompositeZIndex([MODAL_Z_INDEX])
+  const ParentComponent = parentComponent === 'modal' ? Modal : Sheet
+
   return (
     <>
-      <Box
-        display="flex"
-        justifyContent="center"
-      >
+      <Flex alignItems="center" gap={3}>
+        <RadioButton
+          checked={parentComponent === 'modal'}
+          id="modal"
+          label="Open Modal"
+          name="parentComponent"
+          onChange={() => setParentComponent( 'modal' )}
```

Comment:
nit: the spacing around the function arg here is odd

Reply:
DONE